js/client/props-tree.js

```diff
@@ -0,0 +1,334 @@
+  showProps(automationNode, sourceName) {
+    this.lastKey = automationNode.key;
+
+    const remainingProps = this.getPropsToShow(automationNode);
+    const childrenToAdd = [];
+    const propMap = this.getPropMap();
+
+    $('#props-legend-source-name').text(sourceName || '');
+
+    // Organize the properties by intuitive groupings
+    for (let groupName in propMap) {
+      if (!propMap.hasOwnProperty(groupName)) {
+        continue;
+      }
+      let props;
+      if (propMap[groupName] === '*') {
+        // ** Other **
+        // Show remaining properties not in another group
+        props = remainingProps;
+      }
+      else if (propMap[groupName] === '()') {
+        // ** Function **
+        props = Array.from(remainingProps).filter((name) =>
+          automationNode[name].isFunction);
+      }
+      else if (typeof propMap[groupName] === 'string') {
+        // ** String **  -> Remap to a different property
+        props = automationNode[propMap[groupName]];
+      }
+      else {
+        // ** General **  -> Group the set of properties provided
+        props = new Set(propMap[groupName]);
+      }
+
+      // Add props to |group| and remove from |remainingProps|
+      const group = {};
+      for (let prop of props) {
+        if (remainingProps.has(prop)) {   // Only use each prop once
+          group[prop] = automationNode[prop];
+          remainingProps.delete(prop);
+        }
+      }
+
+      // Get view initialization data for the group
+      const groupTreeChildren =
+        this.toTreeData(group);
+      if (groupTreeChildren.length) {
+        childrenToAdd.push({
+          title: groupName,
+          children: groupTreeChildren,
+          expanded: !this.isCollapsedByDefault(groupName)
+        });
+      }
+    }
+
+    this.clearAll();
+    this.getViewRootNode().addChildren(childrenToAdd);
+  }
+
+  // Highlight properties that changed from an treeChange (nodeChanged)
+  highlight(nodeKey, changeData, fadeMs) {
+    if (nodeKey !== this.lastKey) {
+      return;  // This isn't the node that's currently displayed
+    }
+    const highlightImpl = (changeData, keyPrefix) => {
+      for (let prop in changeData) {
+        const viewNode = this.getViewNodeByKey((keyPrefix || '') + prop);
+        if (viewNode) {
+          const change = changeData[prop];
+          if (typeof change === 'object') {
+            // Expand and highlight changed sub-properties like
+            // htmlAttributes.id
+            viewNode.setExpanded(true);
+            highlightImpl(changeData[prop], prop + '.');
+          }
+          else {
+            viewNode.addClass('highlight');
+            // Show what the old property value was
+            viewNode.setTitle(viewNode.title +
+              '<span class="was"> (was ' + change + ')</span>');
+            if (fadeMs) {
+              setTimeout(() => {
+                viewNode.removeClass('highlight');
```

Comment:
How do you want to make this accessible? A timeout isn't
really good for accessibility